src/core/dom/parse-date-attributes.js

```diff
@@ -8,10 +8,16 @@ import {TimestampDef, parseDate} from '#core/types/date';
  * @type {Object<string, DateParserDef>}
  */
 const dateAttrParsers = {
-  'datetime': (datetime) =>
-    userAssert(parseDate(datetime), 'Invalid date: %s', datetime),
-  'end-date': (datetime) =>
-    userAssert(parseDate(datetime), 'Invalid date: %s', datetime),
+  'datetime': (datetime) => {
+    const d = parseDate(datetime);
```

Comment:
🤮 . This exact use case was called out in those TS Issues you linked to

Reply:
Yep =/